app.html

```diff
@@ -1114,6 +1130,75 @@
   }
 }    
 
+var rawHidFilter = { vendorId: 0x16C0, productId: 0x0486, usagePage: 0xFFAB, usage: 0x0200 }; // RawHID interface
+var rawHidParams = { filters: [rawHidFilter] };
+var hid_device;
+var outputReportId = 0x00;
+//incoming rawHID
+function handleInputReport(e) {
+	var text = "";
+	var data_in = [];
+	var i;
+	var len = 0;
+	for (i = 0; i < 64; i++) {
+	  if (e.data.getUint8(i) > 0){	
+	    text += String.fromCharCode(e.data.getUint8(i));
+		len++;
+	  }
+	}
+	OnData(e.data.buffer.slice(0,len));
+};
+async function connectHID() {
+};
+//send buffer rawHID
+function str2ab(str) {
+  var buf = new ArrayBuffer(64);
+  var bufView = new Uint8Array(buf);
+  strLen=str.length;
+  if (strLen < 62) { // 64 -2 for CR-LF
+	  for (var i=0, strLen=str.length; i < strLen; i++) {
+		bufView[i] = str.charCodeAt(i);
+	  }
+	  bufView[i] = 0x0D;
+	  i++
+	  bufView[i] = 0x0A;
+  }
+  return bufView;
+}
+
+var statusInterval; 
```

Comment:
all of thsee variables should have "hid" in them somewhere.

Reply:
These are actually obsolete, so I have removed them